Dockerfile

```diff
@@ -10,6 +10,7 @@ COPY ocrd ./ocrd
 COPY ocrd_modelfactory ./ocrd_modelfactory/
 COPY ocrd_models ./ocrd_models
 COPY ocrd_utils ./ocrd_utils
+RUN mv ./ocrd_utils/ocrd_logging.conf /etc
```

Comment:
What kind of template do you have in mind? Really basic? I'd like to see at least a console and a file handler, since this is a common requirement in prod environments.

Reply:
Foremost, one that reproduces the default you found in the code:
- `INFO` level for most loggers, not just `ocrd.workspace` – your `ERROR` for the root logger would be inherited by processor loggers
- stdout handler only

> I'd like to see at least a console and a file handler, since this is a common requirement in prod environments.

There's nothing wrong with merely showing how to add one by example. But I don't see the point in having a file logger with a fixed file path (`ocrd.log`) by default. This will inevitably cause problems (e.g. multiple processes conflicting in that file, either synchronously or successively).

Reply:
Yes, let's keep the default as minimal as it was before: [log to stdout with a simple pattern](https://ocr-d.de/en/spec/cli#logging) and log level at INFO (which is why the tests fail I think).

Reply:
The test at expects it to be `logging.INFO`but actual it is `logging.DEBUG`. I've got no idea where this is coming from ...

I found just this at `tests/test_workspace.py`, which could(?) address this:
```
from ocrd_utils import setOverrideLogLevel
setOverrideLogLevel('DEBUG')
```
Is this modification set for the whole testing range?

Reply:
> I found just this at `tests/test_workspace.py`, which could(?) address this:
> 
> ```
> from ocrd_utils import setOverrideLogLevel
> setOverrideLogLevel('DEBUG')
> ```
> 
> Is this modification set for the whole testing range?

No just for that test set (`test_workspace`). It simulates a `--log-level DEBUG` run-time parameter.